pkg/api/register.go

```diff
@@ -35,12 +33,6 @@ const GroupName = "descheduler"
 // SchemeGroupVersion is group version used to register these objects
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
 
-func init() {
-	if err := addKnownTypes(scheme.Scheme); err != nil {
```

Comment:
It looks like the `addKnownTypes` function(lines 46 to 51) is no longer called. So should the `addKnownTypes` function just be removed from this file too?

I also noticed that some other files have their own `addKnownTypes` functions, so maybe these should be cleaned up too?
* https://github.com/kubernetes-sigs/descheduler/blob/4014ebad9230f493bcafbdf95aa1c6749a432347/pkg/apis/componentconfig/register.go#L38
* pkg/apis/componentconfig/v1alpha1/register.go
* https://github.com/kubernetes-sigs/descheduler/blob/4014ebad9230f493bcafbdf95aa1c6749a432347/pkg/apis/componentconfig/v1alpha1/register.go#L51

I'm honestly not sure if these need to be cleaned up or not, but just wanted to point this out just in case.

Reply:
Good point, let me check these out. I think at least some of them can be removed if not all

Reply:
This `addKnownTypes` is still used by the `SchemeBuilder`: https://github.com/kubernetes-sigs/descheduler/pull/337/commits/72946602bbc51c3aee3fe438120040a04e53cbdb#diff-f0456cede4d61f48e51015136685ba1dR25

The others in ComponentConfig aren't called, but I think I need to update those to manually invoke `AddToScheme` like I did here